contracts/epochs/FlowClusterQC.cdc

```diff
@@ -299,7 +299,7 @@ pub contract FlowClusterQC {
             let isValid = publicKey.verify(
                 signature: voteSignature.decodeHex(),
                 signedData: voteMessage.decodeHex(),
-                domainSeparationTag: "FLOW-V0.0_Collector-Vote",
+                domainSeparationTag: "FLOW-Collector_Vote-V00-CS00-with-",
```

Comment:
Is there a way to verify that the provided tag is correct? Just want to double confirm.

Reply:
Good question. The tests in the repo are supposed to verify the tag is correct: 
 - the test generates signatures using a tag constant imported from `flow-go` repo (it means the tag used by the protocol).
 - the test submits the signatures to get verified by the contract. 
 - if the there is a tag mismatch, the signature verification fails. 
 
 The constant tag is still not read from `flow-go` yet though, it is hardcoded for now in the tests. Once I update `flow-go` with the new constants, I'll update the tests in this repo again (a circular dependency is the reason behind all these steps)